### PR TITLE
Publish post about Multidimensional Interop tests

### PR DESCRIPTION
**Pull request by MarcoPolo**
This is a copy of https://pl-strflt.notion.site/Multidimensional-Interoperability-6d2cc542309e413a930b4ec233b8cc5e?pvs=4.

Please leave comments in the Notion doc rather than this PR. I'll resolve all comments there before merging.

### CODE REVIEW

**Comment by marten-seemann** [comment]
I just reviewed it in Notion.

**Comment by MarcoPolo** [comment]
All notion comments are resolved. Any other blockers? I'd like to have this in a state where it's ready to launch without me.

**Review by p-shahi — APPROVED** [review]
This looks good to me, let's merge this tomorrow?

File: src/_blog/multidim-interop.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+tags:
+  - libp2p
+title: Testing all the libp2ps
+description: Basic interoperability tests for every libp2p implementation on all transport dimensions.
+date: 2023-08-09
```
Comment (p-shahi) [inline src/_blog/multidim-interop.md:6]:
```suggestion
date: 2023-08-22
```

**Pull request merged**